### PR TITLE
Remove nsq hardcoded names

### PR DESCRIPTION
**Pull request by mbakish**
* changed the names to variables and edit the args accordingly
* didn't changed the ports since they are all clusterIP

### CODE REVIEW

File: nsq/templates/nsqd.yaml [hunk 1]
```diff
@@ -43,11 +43,11 @@ spec:
           value: {{ .Release.Namespace }}
         args:
         - |
-          /nsqd -broadcast-address=$HOSTNAME.nsqd.$(NAMESPACE).svc.cluster.local \
+          /nsqd -broadcast-address=$HOSTNAME.{{ .Values.nsqd.name }}.$(NAMESPACE).svc.cluster.local \
           --max-req-timeout=3h0m0s \
           --max-msg-timeout=1h0m0s \
           {{- range $i, $e := until (int .Values.nsqlookupdReplicaCount) }}
-          --lookupd-tcp-address=nsqlookupd-{{ $i }}.nsqlookupd.$(NAMESPACE).svc.cluster.local:4160 \
+          --lookupd-tcp-address={{ .Values.nsqlookupd.name }}-{{ $i }}.{{ .Values.nsqlookupd.name }}.$(NAMESPACE).svc.cluster.local:4160 \
```
Comment (vitalis) [inline nsq/templates/nsqd.yaml:50]: I think `.Values` not accessible here

Comment (GuyCarmy) [reply nsq/templates/nsqd.yaml:50]: vitaly is correct. that why there is the "NAMESPACE" env var.

File: nsq/templates/nsqd.yaml [hunk 2]
```diff
@@ -30,11 +30,11 @@ spec:
         - name: NAMESPACE
           value: {{ .Release.Namespace }}
```
Comment (GuyCarmy) [inline nsq/templates/nsqd.yaml:31]: add name here an env var, the value can come from the .Values and the env var will exist in the pod (for the "args")

File: nsq/templates/nsqd.yaml [hunk 3]
```diff
@@ -30,11 +30,11 @@ spec:
...
         args:
-        - /nsq_exporter -nsqd.addr=http://$HOSTNAME.nsqd.$(NAMESPACE).svc.cluster.local:4151/stats
+        - /nsq_exporter -nsqd.addr=http://$HOSTNAME.{{ .Values.nsqd.name }}.$(NAMESPACE).svc.cluster.local:4151/stats
```
Comment (GuyCarmy) [inline nsq/templates/nsqd.yaml:33]: .Values won't work here too

File: nsq/templates/nsqd.yaml (hunk 2 above)
Comment (mbakish) [reply nsq/templates/nsqd.yaml:31]: changed to NSQD-NAME and NSQLOOKUPD-NAME

File: nsq/templates/nsqd.yaml (hunk 1 above)
Comment (mbakish) [reply nsq/templates/nsqd.yaml]: fixed

File: nsq/templates/nsqd.yaml (hunk 3 above)
Comment (mbakish) [reply nsq/templates/nsqd.yaml]: fixed

**Review by vitalis — APPROVED** [review]

**Pull request merged**